backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/MembershipImportImpl.java

```diff
@@ -182,4 +184,28 @@ public EList<TextualRepresentation> getTextualRepresentation() {
         return new EcoreEList.UnmodifiableEList<>(this, SysmlPackage.eINSTANCE.getElement_TextualRepresentation(), textualRepresentation.size(), textualRepresentation.toArray());
     }
 
-} //MembershipImportImpl
+    @Override
```

Comment:
Please add @generated NOT in comment for this method or it will be deleted after the next code generation.

Reply:
Done